bbmri/modules/eric-setup.sh

```diff
@@ -6,6 +6,13 @@ if [ "${ENABLE_ERIC}" == "true" ]; then
 
 	# Set required variables
 	ERIC_BROKER_ID=broker.bbmri.samply.de
+	ERIC_ROOT_CERT=eric
+
+	if [ "{$ENABLE_TEST}" == "true" ]; then
```

Comment:
```suggestion
	if [ "${ENABLE_TEST}" == "true" ]; then
```

Is this tested? Because with this typo it shouldn't work